src/main/java/core/mate/academy/producers/ExcavatorProducer.java

```diff
@@ -0,0 +1,14 @@
+package core.mate.academy.producers;
+
+import core.mate.academy.model.Excavator;
+import java.util.List;
+
+public class ExcavatorProducer implements MachineProducer {
+    @Override
+    public List get() {
+        Excavator excavator1 = new Excavator("im123", "red", "pink", 123);
```

Comment:
As I understood from condition of exercise, MachineProducer implementations method get() should return the list of specific machines. For example: `List<Excavator> get()`. I mean, you return simply List, but it should be parameterized.